js/canvasEditor.js

```diff
@@ -1,3 +1,14 @@
+window.onerror = function (msg, url, lineNo, columnNo, error) {
+  if (typeof Raven !== 'undefined') {
+    Raven.captureMessage('Error loading image:  ' + msg + '. At line: ' + lineNo + '. In the file: ' + url, {
```

Comment:
1. If we're attaching this to `window.onerror`, it means that the error doesn't necessarily mean "Error loading image". We used this message in the past because it was attached to `img.onerror`.
1. If we're going to attach something to `window.onerror`, why not attach this to all widget interfaces (or I thought we already have it)?

Reply:
@tonytlwu 
1. I agree, we shouldn't assign this to `window.onerror`
2. Good point, we can update our core JS API to attach such listener on widget interfaces

Reply:
Cool. Let's close this then. I've left a comment in the original issue.

Reply:
PR https://github.com/Fliplet/fliplet-api/pull/3811